targets/ramips/profiles/usb_mt7620/config

```diff
@@ -470,39 +577,51 @@ CONFIG_BUSYBOX_DEFAULT_FEATURE_DU_DEFAULT_BLOCKSIZE_1K=y
 CONFIG_BUSYBOX_DEFAULT_ECHO=y
 CONFIG_BUSYBOX_DEFAULT_FEATURE_FANCY_ECHO=y
 CONFIG_BUSYBOX_DEFAULT_ENV=y
-# CONFIG_BUSYBOX_DEFAULT_FEATURE_ENV_LONG_OPTIONS is not set
 # CONFIG_BUSYBOX_DEFAULT_EXPAND is not set
-# CONFIG_BUSYBOX_DEFAULT_FEATURE_EXPAND_LONG_OPTIONS is not set
+# CONFIG_BUSYBOX_DEFAULT_UNEXPAND is not set
 CONFIG_BUSYBOX_DEFAULT_EXPR=y
 CONFIG_BUSYBOX_DEFAULT_EXPR_MATH_SUPPORT_64=y
+# CONFIG_BUSYBOX_DEFAULT_FACTOR is not set
 CONFIG_BUSYBOX_DEFAULT_FALSE=y
 # CONFIG_BUSYBOX_DEFAULT_FOLD is not set
 CONFIG_BUSYBOX_DEFAULT_FSYNC=y
 CONFIG_BUSYBOX_DEFAULT_HEAD=y
 CONFIG_BUSYBOX_DEFAULT_FEATURE_FANCY_HEAD=y
+# CONFIG_BUSYBOX_DEFAULT_HOSTID is not set
+CONFIG_BUSYBOX_DEFAULT_ID=y
+# CONFIG_BUSYBOX_DEFAULT_GROUPS is not set
 # CONFIG_BUSYBOX_DEFAULT_INSTALL is not set
 # CONFIG_BUSYBOX_DEFAULT_FEATURE_INSTALL_LONG_OPTIONS is not set
+# CONFIG_BUSYBOX_DEFAULT_LINK is not set
 CONFIG_BUSYBOX_DEFAULT_LN=y
 # CONFIG_BUSYBOX_DEFAULT_LOGNAME is not set
 CONFIG_BUSYBOX_DEFAULT_LS=y
 CONFIG_BUSYBOX_DEFAULT_FEATURE_LS_FILETYPES=y
 CONFIG_BUSYBOX_DEFAULT_FEATURE_LS_FOLLOWLINKS=y
 CONFIG_BUSYBOX_DEFAULT_FEATURE_LS_RECURSIVE=y
+CONFIG_BUSYBOX_DEFAULT_FEATURE_LS_WIDTH=y
 CONFIG_BUSYBOX_DEFAULT_FEATURE_LS_SORTFILES=y
 CONFIG_BUSYBOX_DEFAULT_FEATURE_LS_TIMESTAMPS=y
 CONFIG_BUSYBOX_DEFAULT_FEATURE_LS_USERNAME=y
 CONFIG_BUSYBOX_DEFAULT_FEATURE_LS_COLOR=y
 CONFIG_BUSYBOX_DEFAULT_FEATURE_LS_COLOR_IS_DEFAULT=y
 CONFIG_BUSYBOX_DEFAULT_MD5SUM=y
+# CONFIG_BUSYBOX_DEFAULT_SHA1SUM is not set
```

Comment:
Is it ok if I give it as a dependency for ````plugin-gargoyle-usb-storage````?

Reply:
Yes, but make it a dependency of `share_users` instead. `usb-storage` relies on share_users so the dependency will carry through.

Reply:
Hm, now that I think about it, I am not sure how to make this as dependency or provided by share-users. This is not a package, but some setting in menuconfig. What should I write into package/share-users/Makefile ?

Reply:
Ok, I added something which I thought should work based on https://wiki.openwrt.org/doc/devel/dependencies but somehow it does not reflect in the config file. What did I do wrong?

Reply:
I’ve asked for some assistance from the Openwrt devs on this one. 
I’m sure it is doable, but I don’t know.

Reply:
Making the config file bigger is ok.

We either include sha1sum or rewrite everything that relies on it. 
At the time it was easier for me to add it back.

Reply:
Hm, I don't like this option. If we set Customize busybox options, then (apart from having a 30k larger config file) it would mark the build tainted for openwrt. 

I think the long term solution should be to get rid of the sha1sum dependency. 

However, for the short term I think it would still be more clean to patch ````/ramips-src/package/utils/busybox/Config-defaults.in```` to have ````BUSYBOX_DEFAULT_SHA1SUM```` default to ````y```` instead of ````n````. And this would then work for all gargoyle builds, not only the ramips/mt7620. 

TL;DR: I would prefer pathcing ````/ramips-src/package/utils/busybox/Config-defaults.in```` over having a 30k bigger config file. Would this be an agreeable solution?

Reply:
Sure. But you still need to add it to your current config of course